py/stackctrl.c

```diff
@@ -25,6 +25,11 @@
  */
 
 #include "py/runtime.h"
+
+// This API is deprecated, please use py/cstack.h instead
```

Comment:
Suggest putting this comment above the include of py/runtime.h, to mirror `stackctrl.h`.